### ISSUE TITLE
Installation from SourceForge hosted plugins not working

### ISSUE THREAD

**Issue by LarsGit223** [splitbrain/dokuwiki#2410]
On working on the extensions cleanup I noticed that the ticketlinker plugin which is hosted on SourceForge cannot be installed. Clicking on the download link on the plugin page always opened a download page instead of delivering the ```.zip``` archive of the plugin immediately.

This made me contact the SourceForge support and they wrote back that the download is started immediately only if no-browser user-agent header is received and if there is also no Referer header.

So for a test I changed the user-agent in file ```inc/HTTPClient.php``` from
``` php
$this->agent        = 'Mozilla/4.0 (compatible; DokuWiki HTTP Client; '.PHP_OS.')';
```
to
``` php
$this->agent        = 'DokuWiki HTTP Client';
```
After that I could install the ticketlinker plugin successfully.

I know this is not a DokuWiki issue but I thought it's worth posting to discuss if some adjustment should be done or not.

**Comment by splitbrain** [comment]
Interesting observation. It's kinda stupid, but I guess we could change the UA for the extension manager.

**Comment by LarsGit223** [comment]
The failed Travis CI build doesn't seem to be related to my changes.

**Comment by LarsGit223** [comment]
Closed by merging #2417.
